### ISSUE TITLE
Git commit on save

### ISSUE THREAD

**Issue by NickSeagull** [theam/haskell-do#75]
It would be nice to have a save button that automatically `git commit` s the latest changes with the date as the commit message

**Comment by mdibaiee** [comment]
Um, just trying to look from another perspective here: why would we want to do that? I mean, personally I wouldn't want something like that, even if so, I would like to set the message myself 🤔

**Comment by NickSeagull** [comment]
Yeah, maybe the date would be just a placeholder, but take into account that right now the user cannot save their state as the file gets overwrited everytime they press a key.

**Comment by mdibaiee** [comment]
@NickSeagull oops, writing to file on every keypress sounds bad indeed, but a save functionality does not necessarily have to run a `git commit`, I save my code many times before it gets into a shape that becomes _commitable_, why not have both as separate functionalities?
